src/libs/elektra/vheap.c

```diff
@@ -0,0 +1,138 @@
+#include "kdbprivate.h"
+
+/**
+ * Inits the Vheap.
```

Comment:
Also brief should contain a little bit more information than the function name already has.